.changeset/nice-flies-brake.md

```diff
@@ -0,0 +1,25 @@
+---
+"@fluidframework/aqueduct": minor
+"@fluidframework/container-definitions": minor
+"@fluidframework/container-loader": minor
+"@fluidframework/container-runtime": minor
+"@fluidframework/container-runtime-definitions": minor
+"@fluidframework/datastore": minor
+"@fluidframework/devtools-core": minor
+"@fluidframework/fluid-static": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/runtime-utils": minor
+"@fluid-private/test-end-to-end-tests": minor
+"@fluidframework/test-runtime-utils": minor
+"@fluidframework/test-utils": minor
+---
+---
+"section": legacy
+---
+
+The inbound and outbound properties have been removed from IDeltaManager
+
+The inbound and outbound properties were deprecated in a previous release and have been removed from IDeltaManager. Please check pull request [#19636](https://github.com/microsoft/FluidFramework/pull/19636) for alternative APIs.
```

Comment:
I would recommend including the instructions directly here, rather than linking to a PR. That's a lot of navigation steps for our users to jump through to get the information they need.